server/datastore/mysql/migrations/tables/20230520153236_AddHostDepAssignmentsTable.go

```diff
@@ -0,0 +1,30 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230520153236, Down_20230520153236)
+}
+
+func Up_20230520153236(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+          CREATE TABLE host_dep_assignments (
```

Comment:
That table must be added to the list of tables to delete when a host is deleted.